packages/fxa-settings/src/components/ButtonDownloadRecoveryKey/en.ftl

```diff
@@ -16,9 +16,8 @@ recovery-key-file-header = SAVE YOUR ACCOUNT RECOVERY KEY
 # Password resets without this account recovery key can result in data loss.
 recovery-key-file-instructions = Store this file containing your account recovery key in a place you’ll remember. Or print it and keep a physical copy. Your account recovery key can help you recover { -brand-firefox } data if you forget your password.
 
-# { $recoveryKeyValue } is the account recovery key, a randomly generated code in latin characters
-# "Key" here refers to the term "account recovery key"
-recovery-key-file-key-value-v2 = Key: { $recoveryKeyValue }
+# "Key" here refers to the term "account recovery key", a randomly generated code in latin characters
+recovery-key-file-key-value-v3 = Key:
```

Comment:
@bcolsson With this PR, the download file only contains the key unless we add a query parameter to the URL (on local and stage only). With the query param enabled, we can test the detailed file with the key removed from the localized string.

Reply:
Is `in latin characters` accurate? It can contain numbers 0-9 and any letters with the exception of I, L, O, and U. Probably not useful to say it's base32 Crockford 😝, maybe just a `32-character generated code`?

Reply:
`base32 Crockford` might be too specific here, but I've clarified the comment 👍